app/services/marketing_site.rb

```diff
@@ -89,4 +95,18 @@ def self.help_document_capture_tips_url
   def self.security_url
     URI.join(BASE_URL, locale_segment, 'security/').to_s
   end
+
+  def self.help_center_article_url(category:, article:)
+    unless valid_help_center_article?(category: category, article: article)
+      raise ArgumentError.new(
+        "Unknown help center article category: '#{category}', article: '#{article}'",
+      )
+    end
+
+    URI.join(BASE_URL, locale_segment, *['help', category, article].map { |p| "#{p}/" }).to_s
+  end
+
+  def self.valid_help_center_article?(category:, article:)
+    HELP_CENTER_ARTICLES.dig(category)&.include?(article).present?
```

Comment:
I think the `.present?` is redundant, since `.include?` is already truthy/falsy...? even with the nil-safe I think it's still fine

```suggestion
    HELP_CENTER_ARTICLES.dig(category)&.include?(article)
```

Reply:
> I think the `.present?` is redundant, since `.include?` is already truthy/falsy...? even with the nil-safe I think it's still fine

Yeah, it was primarily for the nil-safe operator handling, to convert `nil` to explicitly-`false`. I expect it could probably be fine to just let the return value be truthy/falsey vs. `true`/`false`.

Reply:
Not needed after 99b0b31.